cmd/statusd/main.go

```diff
@@ -73,13 +74,30 @@ var (
 // All general log messages in this package should be routed through this logger.
 var logger = log.New("package", "status-go/cmd/statusd")
 
-func levelFromString(level string) log.Lvl {
-	lvl, err := log.LvlFromString(strings.ToLower(level))
+func enhanceLogger(logger *log.Logger, config *params.NodeConfig) error {
```

Comment:
usually such things are called setupLogger, but really minor thing

Reply:
I know @dshulyak you are right. But the logger was already initialised globally so I thought about this name because it's just adding something to it.